README.md

```diff
@@ -80,6 +80,16 @@ dns_record { "test-1a-record.puppetware.org":
   ttl     => '4800',
 }
 
+# content can also accept array
+
+dns_record { "test-1a-record.puppetware.org":
+  ensure  => present
```

Comment:
Needs a trailing comma to be valid puppet code - which I'm now seeing is also the case with my examples elsewhere in the README.